### PR TITLE
Feat: Simple Copy-Paste Augmentation (Instance Segmentation)

### PR DESCRIPTION
**Pull request by MrinalTyagi**
# Description 

Feature Addition: Simple Copy-Paste Augmentation (Instance Segmentation)
References issue: #28 

POC: @innat @bhack 
Other mentions: @frostbyte012 -> Do look at the implementation template and try adding onto the PR after @innat @bhack approve the basic implementation. 

Comments added for more readability and understanding of @frostbyte012 and will at a later stage if not required. 

Thank for giving me a chance to contribute. :)

### CODE REVIEW

**Comment by innat** [comment]
@MrinalTyagi I can't approve anything here. Only the keras team can. 

( Keep updating your PR until the keras team begins review. )

**Comment by frostbyte012** [comment]
@MrinalTyagi I'm looking into the implementation and will go on with the PR as I understand more about the implementation

**Comment by MrinalTyagi** [comment]
@frostbyte012 You can continue adding to the same pr only rather than creating a new one.

File: keras_cv/layers/preprocessing/copy_paste.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import tensorflow as tf
+from skimage.filters import gaussian
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing/copy_paste.py:2]: yeah, we can't rely on albumerations, skimage, or cv2

**Review by LukeWood — CHANGES_REQUESTED** [review]

**Comment by LukeWood** [comment]
Thank you for your contribution @MrinalTyagi - lets try to get a pure TensorFlow implementation then we can continue with the review.  Thanks!

**Comment by MrinalTyagi** [comment]
@LukeWood Sorry guys for not following up on this as really sick right now. Will soon pick up the task of refining the code to be enabled for merging.

**Comment by LukeWood** [comment]
> @LukeWood Sorry guys for not following up on this as really sick right now. Will soon pick up the task of refining the code to be enabled for merging.

Don't worry about it.  Don't want to be any source of stress.  I appreciate the contribution none the less and follow up when you feel better!

**Comment by LukeWood** [comment]
> @LukeWood Sorry guys for not following up on this as really sick right now. Will soon pick up the task of refining the code to be enabled for merging.

Hey @MrinalTyagi will you be returning to this?  If not,  I'll reopen so that another contributor can take it on.  No pressure or stress!

**Comment by MrinalTyagi** [comment]
> > @LukeWood Sorry guys for not following up on this as really sick right now. Will soon pick up the task of refining the code to be enabled for merging.
> 
> Hey @MrinalTyagi will you be returning to this? If not, I'll reopen so that another contributor can take it on. No pressure or worries! Just want to make sure we move forward on this issue at some point!

Sorry for delay. I'm back on it. :)

**Comment by MrinalTyagi** [comment]
> > > @LukeWood Sorry guys for not following up on this as really sick right now. Will soon pick up the task of refining the code to be enabled for merging.
> 
> > 
> 
> > Hey @MrinalTyagi will you be returning to this? If not, I'll reopen so that another contributor can take it on. No pressure or worries! Just want to make sure we move forward on this issue at some point!
> 
> 
> 
> Sorry for delay. I'm back on it. :)
> 
> 

Hi team. Got some issue with my laptop and have given it for repair. Please assign it to someone else. Sorry for being a blocker.

**Comment by frostbyte012** [comment]
@LukeWood , and @innat I'm trying this one soon next ! Looks like I'll create another new PR for this issue #28 instead of carrying on with this one.

**Comment by LukeWood** [comment]
Closing as we haven't had any activity here and this depends on packages we can't use.

**Comment by BuddhiAbeyratne** [comment]
I understand not shipping this in Keras 
but is it possible for it to be a separate package or something @MrinalTyagi  maybe

**Pull request closed without merging**